charts/telegraf/templates/configmap.yaml

```diff
@@ -20,6 +20,10 @@ data:
       [[outputs.health.contains]]
         field = "buffer_size"
     {{- end }}
+    {{- if .Values.config.inputs_override_config }}
```

Comment:
So a question about this approach.

Is there a reason why this is input specific?  This really could be any part of the Telegraf config, no?  global, input, output, processor, aggregator??

Perhaps we should alter this to be additional_config and have it appended at the end of the configmap?

Reply:
This change was the minimal change to support my use case. If you'd rather have a different implementation, let me know and I will try to update it. In my view, the attempt to convert a dict to the TOML cannot work correctly, so I would even suggest to deprecate it and simply replace it with a (multi-line) string of the TOML. The code in _helpers.tpl for define "inputs" is way too complicated and brittle, I think. I have not looked at the other sections, but I suspect the case can be made there as well.